source/lib/JMSLab/tests/test_autofill.py

```diff
@@ -1,25 +1,64 @@
-import sys
-
-sys.path.append('./source/lib')
-
 from unittest import main, TestCase
-from JMSLab.autofill import GenerateAutofillMacros
+import tempfile, shutil 
 from os.path import exists
 
-
+from ..autofill import GenerateAutofillMacros
 
 class Test(TestCase):
+    
+    def setUp(self):
+        self.tempdir = tempfile.mkdtemp() 
+        self.outfile = self.tempdir + r"output_macros.tex"
+        return
```

Comment:
I don't understand this return statement. Did you mean this?

```suggestion
        return None
```

There are similar 'lonely' return statements below.

Reply:
Trying out the code suggestions here @jmshapir. So cool!

Reply:
Addressed remaining lonely return statements in [e6fc892](https://github.com/JMSLab/Template/pull/57/commits/e6fc89248dfdc62f74fbb214b74f33bd346e4dc8).